src/lib/Scenes/Artwork/Components/__tests__/CommercialEditionSetInformation-tests.tsx

```diff
@@ -0,0 +1,58 @@
+import { Theme } from "@artsy/palette"
+import { mount } from "enzyme"
+import React from "react"
+import { TouchableWithoutFeedback } from "react-native"
+import { CommercialEditionSetInformation } from "../CommercialEditionSetInformation"
+
+const artwork = {
+  edition_sets: [
+    {
+      id: "RWRpdGlvblNldDo1YmJiOTc3N2NlMmZjMzAwMmMxNzkwMTM=",
+      internalID: "5bbb9777ce2fc3002c179013",
+      is_acquireable: true,
+      is_offerable: true,
+      sale_message: "$1",
+      edition_of: "",
+      dimensions: {
+        in: "2 × 2 in",
+        cm: "5.1 × 5.1 cm",
+      },
+    },
+    {
+      id: "RWRpdGlvblNldDo1YmMwZWMwMDdlNjQzMDBhMzliMjNkYTQ=",
+      internalID: "5bc0ec007e64300a39b23da4",
+      is_acquireable: true,
+      is_offerable: true,
+      sale_message: "$2",
+      edition_of: "",
+      dimensions: {
+        in: "1 × 1 in",
+        cm: "2.5 × 2.5 cm",
+      },
+    },
+  ],
+}
+
+describe("CommercialEditionSetInformation", () => {
+  it("changes displays first edition price", () => {
+    const component = mount(
+      <Theme>
+        <CommercialEditionSetInformation setEditionSetId={() => null} artwork={artwork as any} />
+      </Theme>
+    )
+
+    expect(component.html()).toContain("$1")
+  })
+
+  it("changes display price to selected edition set", () => {
```

Comment:
This likely belongs in the test of a higher-level component, but two cases that we should be sure to test are:
- When you have one edition set and the work is enabled for buy now/make offer, when you click one of these buttons, we send the edition set ID with our input payload
- When you have multiple edition sets, when you select one of these, that's the ID that gets sent with the input payload.